core/backend/src/IModelSchemaLoader.ts

```diff
@@ -17,10 +41,14 @@ import { IModelDb } from "./IModelDb";
  * @alpha
  */
 export class IModelSchemaLoader {
-  private _context: SchemaContext;
+  private _context: typeof SchemaContext;
 
   /** @internal */
   public constructor(private _iModel: IModelDb) {
+    // If the ecschema-metadata package was not installed inform the caller.
+    if (metadata === undefined)
+      throw new IModelError(IModelStatus.NotFound, "IModelSchemaLoader requires that @bentley/ecschema-metadata be installed.");
```

Comment:
also list core quantity

Reply:
Right now ecschema-metadata has 2 peer dependencies or perhaps more in the future.  Do we want to list them all here?  The user will get a warning upon installation if they don't have the peers installed.

Reply:
OK